### PR TITLE
chore: schedule daily generateResources action

### PR DESCRIPTION
**Pull request by iProdigy**
Follow-up to https://github.com/pajlads/DinkPlugin/pull/425#discussion_r1495015187

### CODE REVIEW

**Comment by iProdigy** [comment]
some docs to check whether the syntax is correct https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#use-case-create-a-pull-request-to-update-x-periodically

File: .github/workflows/update-resources.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Execute Resource Generator
+        run: ./gradlew generateResources
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          title: "chore: update rarity data from wiki"
```
Comment (Felanbird) [inline .github/workflows/update-resources.yml:25]: can we use `${{ steps.date.outputs.date }}` here so the PRs are at least a little distinguishable

Comment (pajlada) [reply .github/workflows/update-resources.yml:25]: would need the appropriate step added to make this output

Comment (iProdigy) [reply .github/workflows/update-resources.yml]: Done in https://github.com/pajlads/DinkPlugin/pull/452/commits/190d6e1e7d1bce55458c765ea53f4b27e59d8095

**Review by pajlada — APPROVED** [review]

**Review by Felanbird — APPROVED** [review]

**Pull request merged**